README.md

```diff
@@ -40,54 +40,101 @@ git clone https://github.com/fortran-lang/stdlib
 cd stdlib
 ```
 
+
 ### Requirements
 
-The preprocessor ```fypp``` (https://github.com/aradi/fypp) is needed because metaprogramming is used.
-It can be installed using the command line installer ```pip```.
+To build the Fortran standard library you need
+
+- a Fortran 2008 compliant compiler, better a Fortran 2018 compliant compiler
+  (GCC Fortran or Intel Fortran compilers are known to work for stdlib)
+- CMake version 3.14 or newer (alternatively make can be used)
+- a build backend for CMake, like make or ninja (the latter is recommended on Windows)
+- the [``fypp``](https://github.com/aradi/fypp) preprocessor (used as meta-programming tool)
```

Comment:
Super minor style nit-pick, I think we should use `monospace` only for actual code/commands, and not for proper nouns.
```suggestion
- the [fypp](https://github.com/aradi/fypp) preprocessor (used as meta-programming tool)
```